lang/py/src/avro/io.py

```diff
@@ -280,6 +296,64 @@ def read_utf8(self):
     """
     return unicode(self.read_bytes(), "utf-8")
 
+  def read_date_from_int(self):
+    """
+    int is decoded as python date object
+    int stores the number of days from
+    the unix epoch, 1 January 1970 (ISO calendar).
+    """
+    days_since_epoc = self.read_int()
+    return datetime.date(1970, 1, 1) + datetime.timedelta(days_since_epoc)
```

Comment:
not sure i understand this, python date object has no timezone associated to it. Even [java](https://github.com/Yelp/avro/blob/trunk/lang/java/avro/src/main/java/org/apache/avro/data/TimeConversions.java#L31) date does not have TZ info